plugins/contact-resources/src/components/UserInfo.svelte

```diff
@@ -30,17 +29,14 @@
 
   const client = getClient()
   const hierarchy = client.getHierarchy()
```

Comment:
We should not hold hierarchy in component, since it will be not updated on reconnect-refresh.